From c4d9b57b5b0b98abb42c644c72a32e3d691398f9 Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sun, 28 Sep 2014 13:55:01 +0800
Subject: [PATCH 0089/5242] MLK-9618-1 doc: usb: chipidea: select gadget
 drivers for otg compliance test

commit  e8efaab10323e6e36cf558b79a9dfe4cd17a3dfa from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds guide for selecting available gadget drivers for otg and EH
compliance tests.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 520cac9e4fe938887dd45b5b4df6c8e35e125a59)
(cherry picked from commit 6534fab0bd77be544629ac04d4b8bb677d0a3aec)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/usb/chipidea.txt |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/usb/chipidea.txt b/Documentation/usb/chipidea.txt
index 7e385dc..8a9fecc 100644
--- a/Documentation/usb/chipidea.txt
+++ b/Documentation/usb/chipidea.txt
@@ -77,6 +77,14 @@ cat /sys/kernel/debug/ci_hdrc.0/registers
 "On-The-Go and Embedded Host Supplement to the USB Revision 2.0 Specification
 July 27, 2012 Revision 2.0 version 1.1a"
 
+1.4 OTG compliance test
+----------------------
+Only below 3 popular gadget drivers are declared to be USB OTG and EH 2.0
+compliant(with otg descriptor comply with USB OTG and EH 2.0 as a peripheral):
+- mass storage
+- ether
+- serial
+
 2. How to enable USB as system wakeup source
 -----------------------------------
 Below is the example for how to enable USB as system wakeup source
-- 
1.7.9.5

